cmd/meroxa/flink/deploy.go

```diff
@@ -64,6 +69,31 @@ func GetIRSpec(ctx context.Context, jarPath string, secrets map[string]string, l
 	}
 
 	spec.Secrets = secrets
+	// workarounds to validate spec
```

Comment:
Created a task to follow up on this once the workarounds are no longer needed https://github.com/meroxa/product/issues/895

Reply:
thank you.  that should go under this epic https://github.com/meroxa/product/issues/868